### ISSUE TITLE
Type references in definitions

### ISSUE THREAD

**Issue by andry-tino** [andry-tino/Rosetta#3]
Problems might arise when having, for example:

```
namespace A.B {
  public class MyClass : TypeA { ... }
}
```

Where:

```
[ScriptNamespace("NewNamespace")]
public class TypeA { ... }
```

### Suggested approach
Possibility to use transformation to always include the full type name in the syntax before starting the translation process.

**Comment by andry-tino** [comment]
Duplicate of #22 partially solved in #19
